### DISCUSSION TITLE
Provide bottom-level package

### DISCUSSION THREAD

**Discussion by Canglangzi** [FirstGearGames/FishNet#739] [Feature Request]
Sometimes I don't want to use broadcasting. In some cases, I need to initialize all the information of the entire world, so I want to use a data packet. I see that fishnet itself does not provide packet sending. I hope it can provide it. I may also need to implement some network synchronization position issues myself. The current Fishnet's NetworkTransform cannot be extended or it is difficult. The new version of the child synchronization seems to have problems. I hope to provide the underlying packet sending. RPC will send all the data you write in. In addition, there are many scene physics synchronizations in my game. Fishnet does not have these basic columns such as NetworkRigidbody components, etc.

**Comment by FirstGearGames** [comment]
This is too vague for me to address anything specific. Can you please make new discussions for each feature you are asking about, thank you!
